src/v2/Apps/Fair/Routes/FairExhibitors.tsx

```diff
@@ -63,6 +66,7 @@ export const FairExhibitorsFragmentContainer = createFragmentContainer(
   {
     fair: graphql`
       fragment FairExhibitors_fair on Fair {
+        slug
```

Comment:
**suggestion**: Instead of using the fair slug to generate a show URL, I think we can use the `profileID` from the exhibitor which represents the show:

```graphql
{
  fair(id: "the-armory-show-2021") {
    exhibitorsGroupedByName {
      exhibitors {
        profileID
        slug
      }
    }
  }
}
```

`/show/${exhibitor.profileID}`

Reply:
Thanks for the comment, I'll fix it soon 👍